views/main/play.ejs

```diff
@@ -1,32 +1,31 @@
 <% layout('layouts/boilerplate')%>
-
-<div class="container d-flex justify-content-center align-items-center mt-5">
-  <div class="row">
-    <div class="col-sm-8 offset-sm-2 col-lg-6 offset-lg-3">
-      <div class="card shadow">
-        <img
-          src="https://res.cloudinary.com/dyguovdbc/image/upload/w_40/v1676908287/pixelify/placeholder-image_ykgw2b.jpg"
-          alt=""
-          width="800"
-          style="image-rendering: pixelated"
-          class="card-img-top"
-        />
-        <div class="card-body pt-5">
-          <h5 class="card-title">What form of game is this game?</h5>
-          <form>
-            <div class="mb-3">
-              <input
-                class="credentials-input"
-                type="text"
-                id="gamename"
-                name="gamename"
-                autofocus
-                required
-              />
-            </div>
-            <button class="btn btn-success btn-block">Next</button>
-          </form>
-        </div>
+<div class="card">
+  <div class="card-box">
+    <div class="image-box">
+      <img
+        class="card-image"
+        id="pixelatedImage"
+        src="<%=image%>"
+        alt="pixelated image"
+      />
+    </div>
+    <div>
+      <div class="credentials-box">
+        <h5 class="card-title">What form of game is this game?</h5>
+        <form action="/login" method="POST" class="validated-form" novalidate>
```

Comment:
Can you tell me how this functionally going to look like? I would recommend to ditch the `form` element and just place an input because I don't see why you should send a form anywhere unless you plan on using that data somewhere in the long run. 

My first thought would be that you could remove `form` and keep the `input` element. Passing an onclick function to the button you could validate the answer from the input field.